tests/wasmfs/wasmfs_opfs.c

```diff
@@ -118,6 +118,11 @@ int main(int argc, char* argv[]) {
   assert(err == 0);
   emscripten_console_log("closed file");
 
+  err = stat("/opfs/working/foo.txt", &stat_buf);
+  assert(err == 0);
+  assert(stat_buf.st_size == 0);
```

Comment:
Maybe better to test on a file with a non-zero size, to check the output size too?

Reply:
Sounds good. I changed the previous truncate to go to size 1 instead of size 0.